nix/kore-ghc923.nix.d/default.nix

```diff
@@ -11,13 +11,11 @@
       "direct-sqlite" = (((hackage.direct-sqlite)."2.3.26").revisions).default;
       "witherable" = (((hackage.witherable)."0.4.2").revisions).default;
       "witherable-class" = (((hackage.witherable-class)."0").revisions).default;
+      "ghc-events" = (((hackage.ghc-events)."0.17.0.3").revisions).default;
       "tasty-test-reporter" =
         (((hackage.tasty-test-reporter)."0.1.1.4").revisions).default;
       "junit-xml" = (((hackage.junit-xml)."0.1.0.0").revisions).default;
       "compact" = (((hackage.compact)."0.2.0.0").revisions).default;
-      "decision-diagrams" =
-        (((hackage.decision-diagrams)."0.2.0.0").revisions).default;
-      "ghc-events" = (((hackage.ghc-events)."0.17.0.3").revisions).default;
```

Comment:
... so, the rematerialise magic seems to disagree here :man_shrugging:

Reply:
you need to add `decision-diagrams` to `stack-nix-ghc9.yaml` first. Nix uses that file to generate [nix/kore-ghc923.nix.d/default.nix](https://github.com/runtimeverification/haskell-backend/pull/3183/files/0e1cdfe876ceac0878152847ad964c9f209fa7d1..d0b741575f30005de2a823d85fe7b51183cec10b#diff-abe8949cc72bbf41512be940aff6674932710259a4e4b3431380ee805826bf23)